.mailmap

```diff
@@ -0,0 +1,15 @@
+Alexander A. Klimov <alexander.klimov@icinga.com> <alexander.klimov@netways.de>
+Eric Lippmann <eric.lippmann@icinga.com>
```

Comment:
I am not aware that entries with only one address are required. Are they?

Reply:
According to the [Git documentation](https://git-scm.com/docs/gitmailmap), this is allowed, but not required.